data/prod/igbo/frontpage/index.json

```diff
@@ -9,9 +9,9 @@
     "type": "IDX",
```

Comment:
Howcome the data update?

Reply:
I was getting different values from Ares. Some labels were missing in the old fixture files